service/src/main/java/org/broadinstitute/listener/relay/Utils.java

```diff
@@ -69,11 +74,11 @@ public static boolean isValidOrigin(String origin, CorsSupportProperties corsSup
     String finalStringToCheck = stringToCheck;
     return origin.isEmpty()
         || corsSupportProperties.validHosts().stream()
-        .anyMatch(
-            validHost ->
-                Pattern.matches(
-                    validHost.replace(".", "\\.").replace("*", ".*").replace(" ", ""),
-                    finalStringToCheck));
+            .anyMatch(
```

Comment:
None of our refererConfigs in any environment I'm aware of actually use wildcard matching other than the single wildcard alone (`"*"`), and if included it will break Leonardo (pending https://broadworkbench.atlassian.net/browse/IA-4368). I think that this should be removed and the solo wildcard handled specifically.